orders/models.py

```diff
@@ -71,3 +71,24 @@ def cancel_order(self):
 
     def get_order_total_price(self):
         return self.price_per_day * (self.end_date - self.start_date).days
+
+    def get_order_status(self):
+        if self.status == StatusOptions.Pending:
+            return "Pending"
+        elif self.status == StatusOptions.Approved:
+            return "Approved"
+        elif self.status == StatusOptions.Canceled:
+            return "Canceled"
+        elif self.status == StatusOptions.OnGoing:
+            return "On going"
+        return "Finished"
+
+    def is_the_order_cancelable(self):
+        if self.status != StatusOptions.OnGoing or self.status != StatusOptions.Finished:
+            return True
+        return False
+
+    def is_the_order_approvable(self):
+        if self.status != StatusOptions.Pending:
+            return True
+        return False
```

Comment:
this method is only for the daycare user, right? I think you also need to check the capacity of the dates of the order and check if there is a place for more dogs on those dates.